library/src/webapp/editor/ckextraplugins/researcher-training-tool/js/select-multiple-values.js

```diff
@@ -8,11 +8,16 @@ CKEDITOR.ui.dialog.uiElement.prototype.getValues = function() {
   // get the options from the dialog's input element
   var select = this.getInputElement();
   var selectedOptions = select.$.selectedOptions;
+  if (isIE()){
```

Comment:
Doesn't need changing, but it's useful to comment why this hack is here.
Hacks for browsers are tricky to understand when you come back to them in 6 months time.